documentation/api/parameters.md

```diff
@@ -90,8 +93,6 @@ The [examples](#examples) include a mixture of `UserCode` and `SystemCode` to he
 The following logger categories are used inside the target application when capturing parameters:
 | Category Name | Description |
 | -- | -- |
-| `DotnetMonitor.ParameterCapture.UserCode` | Parameters captured in methods considered `UserCode`. |
```

Comment:
Also remove the `## UserCode vs SystemCode` section from the documentation since it no longer applies